exercises/handling-errors/practice/src/client.ts

```diff
@@ -0,0 +1,76 @@
+import { Connection, Client } from '@temporalio/client';
+import { pizzaWorkflow } from './workflows';
+import { Pizza, PizzaOrder, TASK_QUEUE_NAME } from './shared';
+
+async function run() {
+  const connection = await Connection.connect({ address: 'localhost:7233' });
```

Comment:
Optional: Since you are doing some narration, it might be okay to comment something along the lines "Set up the Temporal Client connection"  "Establish the pizza order Workflow"